lib/src/command/downgrade.dart

```diff
@@ -27,17 +27,17 @@ class DowngradeCommand extends PubCommand {
         negatable: false,
         help: "Report what dependencies would change but don't change any.");
 
-    argParser.addFlag('packages-dir',
-        negatable: true,
-        help: "Generate a packages/ directory when installing packages.");
+    argParser.addFlag('packages-dir', negatable: true, hide: true);
   }
 
   Future run() async {
+    if (argResults.wasParsed('packages-dir')) {
+      log.warning(
+          log.yellow('The --packages-dir flag is an unsupported no-op'));
```

Comment:
"no-op" is jargon. How about:

> The --packages-dir flag is no longer used and does nothing.

Here and below.

Reply:
Good call. Done